src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -66,11 +70,19 @@ internal WebAssemblyHostBuilder(WebAssemblyJSRuntimeInvoker jsRuntimeInvoker)
             };
         }
 
+        private void InitializeNavigationManager(WebAssemblyJSRuntimeInvoker jsRuntimeInvoker)
```

Comment:
I find this a bit more intuitive. @SteveSandersonMS what do you feel?

Reply:
This all looks good to me now.